attestation-service/attestation-service/src/rvps/mod.rs

```diff
@@ -55,10 +56,24 @@ impl Default for RvpsConfig {
     }
 }
 
+#[derive(Error, Debug)]
+pub enum RvpsError {
+    #[error("Failed creating rvps service: {0}")]
+    CreateRvps(String),
+    #[error("feature `rvps-grpc` or `rvps-builtin` should be enabled")]
+    FeatureNotEnabled,
+    #[error("Serde Json Error")]
+    SerdeJson(#[from] serde_json::Error),
+    #[error("Returned status: {0}")]
+    Status(#[from] tonic::Status),
+    #[error("tonic transport error: {0}")]
+    TonicTransport(#[from] tonic::transport::Error),
```

Comment:
I don't think those `...: {0}` will do anything, there's no embedded string, see thiserror docs:

<img width="462" alt="image" src="https://github.com/confidential-containers/kbs/assets/273280/35592589-85a3-4b47-8744-bc12a46cfd3c">

If you wrap an error you don't need string templating, thiserror will do this.